config/settings/base.py

```diff
@@ -412,7 +410,10 @@
 # String for overriding how much time will be needed for the task's review.
 SPRINT_REVIEW_DIRECTIVE = fr"\[~{JIRA_BOT_USERNAME}\]: plan (\d+) hours for reviewing this task"
 # Regex for extracting sprint number from the name of the sprint.
-SPRINT_NUMBER_REGEX = env.str("SPRINT_NUMBER_REGEX", r"Sprint (\d+)")
+# It is also used for distinguishing standard sprints from special ones (e.g. Stretch Goals).
+# TODO: Uncomment this after ending the current sprint.
+# SPRINT_NUMBER_REGEX = env.str("SPRINT_NUMBER_REGEX", r"\w+\.(\d+)")
+SPRINT_NUMBER_REGEX = env.str("SPRINT_NUMBER_REGEX", r"\w+.*?(\d+)")
```

Comment:
Given this REGEX works for both the old and new styles, can't this be kept as the default and the more strict be set in the env? This would remove the need for the above TODO.